fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkcommon/Constants.java

```diff
@@ -30,7 +30,9 @@
     public static final byte[] NDJSON_LINESEPERATOR = "\r\n".getBytes();
 
     public static final int IMPORT_MAX_PARTITIONPROCESSING_THREADNUMBER = 10;
```

Comment:
At the top of this class, please remove the comment above the package declaration. 

```
/**
 * Constants for the JavaBatch Jobs.
 * Constants for the JavaBatch Jobs.
 *
 *
 */
 */
```

Reply:
done

Reply:
Albert, I think you removed the wrong one.  We want to keep the javadoc on the class, but remove the extra one that was in the wrong place.

Reply:
Right - not about this comment - it was at the top of the class (I scanned the copyrights and found it)

Reply:
ha, done! thanks!